index.html

```diff
@@ -1,20 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="css/text" href="style.css">
-    <title>Document</title>
+    <title>PokeYou</title>
 </head>
+
 <body>
 
-    <form>
-        <input type="text" id="pokeMon">
-        <input type="submit" id="pokeSubmit">
-    </form>
+    <!-- Screen 1 HTML -->
+
+    <!-- Addition of possible Jumbotron to add our TITLE -->
+
+    <!-- Catchy text that brings catches the user's attention-->
+    <div>
+        <h1>POKEYOU</h1>
+    </div>
+    <div>
+        <p id="Catchy">THIS WILL BE SOMETHING CATCHY</p>
+    </div>
+<br>
+<br>
+<br>
+    <!-- Screen 2 HTML -->
+
+    <label for="avatar">Choose a profile picture:</label>
```

Comment:
The image and image submission button should be in a div. I image the styling for this will be separate from the pokemon returned and stats so we want to be able to control this as a separate entity.